cypress/integration/application/index.js

```diff
@@ -1,5 +1,5 @@
 import config from '../../support/config/services';
-import appConfig from '../../../src/testHelpers/serviceConfigs';
+import appConfig from '../../../src/server/utilities/serviceConfigs';
```

Comment:
Can't this line also make use of the webpack alias you made? Or does cypress not like that? 
LGTM besides this, nice work!

Reply:
Cypress runs on a different set of configurations, using aliases here will lead to unresolved imports.

Unless we configure Cypress to use our webpack configuration. I'm not sure if that's a good idea.

Reply:
Ah I see, I didn't know that, thanks! I'll approve it now